### DISCUSSION TITLE
【上线其他实例】和【查看提交信息】

### DISCUSSION THREAD

**Discussion by HanochLzd** [hhyo/Archery#1663] [Q&A] [answered]
没有发现【上线其他实例】和【查看提交信息】这两个按钮的权限配置，是只有超级管理员权限才有吗？

![image](https://user-images.githubusercontent.com/26274623/177912367-fb030d1a-4443-49e7-a380-1971895bc7ae.png)

**Comment by nick2wang** [comment, marked as answer]
上线其他实例：只允许发起人提交其他实例
查看提交信息：超级管理员/是否可审核/是否可执行/是否可回滚（ 具体定义可以看这里：https://github.com/hhyo/Archery/blob/master/sql/views.py ）
https://github.com/hhyo/Archery/blob/2ffb6de1af283d0201a38aa5d29bc28f02a6d767/sql/templates/detail.html#L10-L16
